### PR TITLE
Added Content

### PR DESCRIPTION
**Pull request by prerana1821**

### CODE REVIEW

File: comp.html [hunk 1]
```diff
@@ -0,0 +1,968 @@
...
+                    <h3 class="secondary-txt-showcase" >@size avatars</h3>
+                    <div class="row-av">
+                        <img class="avatar av-sm" src="images/person-1.jpg" alt="">
+                        <img class="avatar " src="images/person-2.jpg" alt="">
+                        <img class="avatar av-lg" src="images/person-3.jpg" alt="">
+                    </div>
+
+                    <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=panda-syntax&wt=sharp&l=css&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=15px&lh=128%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522avatar%2520av-sm%2522%2520src%253D%2522...%2522%2520alt%253D%2522..%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2522%2520src%253D%2522...%2522%2520alt%253D%2522..%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520av-lg%2522%2520src%253D%2522...%2522%2520alt%253D%2522..%2522%253E"
+                    style="width: 445px; height: 102px; border:0; transform: scale(1); overflow:hidden;margin: 2rem 0 1rem; border-radius: 1rem;"
```
Comment (poojasabnani) [inline comp.html]: The common styling in iframe component can be moved to a class as it's getting reused

File: comp.html [hunk 2]
```diff
@@ -0,0 +1,968 @@
...
+                    style="width: 490px; height: 380px; border:0; transform: scale(1); overflow:hidden;border-radius:1rem;margin:2rem 0"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                                    </div>
```
Comment (poojasabnani) [inline comp.html]: small change: this can be formatted correctly

File: comp.html [hunk 3]
```diff
@@ -0,0 +1,968 @@
...
+                    style="width: 544px; height: 231px; border:0; transform: scale(1); overflow:hidden;margin:2rem;border-radius:1rem;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+
+                    <h3 class="secondary-txt-showcase">@Horizontal Cards</h3>
+                    <div class="column">
+                        <div class="card-horztl">
```
Comment (poojasabnani) [inline comp.html]: Optional: a clearer name for class can be used for user friendliness as card-horizontal or card-h only

File: comp.html [hunk 4]
```diff
@@ -0,0 +1,968 @@
...
...20%2520%253Cp%253EDescription..%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Csmall%253EThis%2520is%2520a%2520small%2520text%253C%252Fsmall%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253C%252Fli%253E%250A%2520%2520%253Cli%2520class%253D%2522stacked-list-item%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522stacked-list-item-ic%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-lg%2520fa-lightbulb%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522list-content%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%253EThis%2520is%2520a%2520list%2520heading%25201%253C%252Fh3%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253EDescription..%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Csmall%253EThis%2520is%2520a%2520small%2520text%253C%252Fsmall%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253C%252Fli%253E%250A%253C%252Ful%253E%2520"
```
Comment (poojasabnani) [inline comp.html]: this can be formatted

File: comp.html [hunk 5]
```diff
@@ -0,0 +1,968 @@
...
...2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge-icn%2520yellow%2520bdg-top%2522%253E20%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar%2520av-yellow%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-lg%2520fa-shopping-cart%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522images%252Fperson-1.jpg%2522%2520class%253D%2522avatar%2522%2520alt%253D%2522%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Ful%253E%250A%2520%2520%2520%2520%253C%252Fnav%253E%250A%253C%252Fheader%253E"
```
Comment (poojasabnani) [inline comp.html]: here as well

File: components.css [hunk 6]
```diff
@@ -0,0 +1,1029 @@
...
+  --secondary-box-shadow: 2px 4px 20px -1px rgb(0 0 0 / 3%),
+    -1px -2px 20px -1px rgb(0 0 0 / 3%);
+  --tertiary-box-shadow: 6.61px 6.61px 9px #e6eef5, -6.61px -6.61px 9px #faffff;
+}
+
+body {
+  font-family: "Varela Round", sans-serif;
+}
+
+/* Utils
```
Comment (poojasabnani) [inline components.css]: commented out code can be cleaned up

File: components/Badges/badge.css [hunk 7]
```diff
@@ -0,0 +1,35 @@
+/* badges */
+.badge-av {
+  position: relative;
+}
+
+.badge-icn {
```
Comment (poojasabnani) [inline components/Badges/badge.css]: naming can be made more consistent: either badge- or bdg-

File: components/Buttons/button.css [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-shadow: var(--primary-box-shadow);
+  outline: none;
+  border: none;
+}
+
+.flt-pri {
```
Comment (poojasabnani) [inline components/Buttons/button.css]: here as well naming can be made more consistent. Either float- or flt- .

File: components/Cards/card.css [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+    120px 20px,
+    90px -10px,
+    30px -10px,
+    0px 20px,
+    10px 30px,
+    110px 30px
+  );
+}
+
+.bdge-cl {
```
Comment (poojasabnani) [inline components/Cards/card.css]: naming consistency can be considered here as well. 3 different names are bdg- , badge-, bdge-

File: components/Ratings/ratings.css [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+
+.checked {
+  color: var(--dk-yellow);
+}
+
+.bdge-ratings {
```
Comment (poojasabnani) [inline components/Ratings/ratings.css]: here as well

File: components/Ratings/ratings.html [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ratings</title>
+    <link rel="stylesheet" href="../Util/util.css" />
+    <link rel="stylesheet" href="./ratings.css" />
+    <link rel="stylesheet" href="../mediaqueries.css" />
+  </head>
+  <body>
+    <section class="sect">
```
Comment (poojasabnani) [inline components/Ratings/ratings.html:13]: here as well. some place its sections and here sect. It can be made consistent

File: resources.html [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+      <div class="logo">
+        <a href="./index.html"
+          ><img src="./images/preCodes.png" alt="Logo"
+        /></a>
+        <a href="./index.html"><p class="logo-txt">preKit</p></a>
+      </div>
+
+      <ul>
+        <li><a href="#typography">Coming Soon</a></li>
+        <!-- <li><a href="#colors">Colors</a></li> -->
```
Comment (poojasabnani) [inline resources.html]: commented code can be removed.

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,271 @@
...
+  top: 0;
+  left: 0;
+  padding: 1.3rem 1rem 0 2rem;
+  background-color: #ffffff;
+}
+
+body > main {
+  grid-area: main;
+  border-radius: 1.5rem 0 0 0;
+  padding: 2rem 10% 0;
```
Comment (poojasabnani) [inline styles.css:39]: this can be made consistent without mixing rem and % . 1 unit can be used for consistency

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,271 @@
...
+}
+
+.showcomp {
+  padding: 1rem 3rem;
+}
+
+.column-av {
+  display: flex;
+  flex-direction: column;
+  /* align-items: center; */
```
Comment (poojasabnani) [inline styles.css]: this can be removed.

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,271 @@
...
+.comp-nav ul li a {
+  font-size: large;
+  color: #000;
+}
+
+.comp-nav ul li a {
+  width: 100%;
+}
+/* 
+.comp-nav ul li a:hover:before {
```
Comment (poojasabnani) [inline styles.css]: this can be also removed

**Review by poojasabnani — COMMENTED** [review]
Hey @prerana1821 ,

Great code base with a lot of detailing. I've just suggested in review for maintaining naming consistencies and code clean up with commented out code.

**Review by purvasheth — COMMENTED** [review]
Hi @prerana1821,
Impressed 💯 
Your library is so pretty!

Somethings I noticed
+  text contrast is less than it should be almost everywhere. Even on primary buttons. Please look into this. Your color sense is amazing! Keep the contrast in mind though.
+ your navigation can have an active style. Both top and side. The hover effect is pretty cool. But once the user selects something, it would be nice if he/she knows what is selected. 

Pooja covered all the code-level comments :)

File: comp.html (hunk 1 above)
Comment (prerana1821) [reply comp.html]: Thanks a lot Pooja for all your suggestions, actually for iframe I have different styling for all different styles.

**Comment by prerana1821** [comment]
Hey @py93 Once again, Thanks a lot Pooja, for all your suggestions, this means a lot alot to me. I have done all the changes that you have suggested. Just one question, for code reviewing do you clone the project or do it via github only?

File: comp.html (hunk 1 above)
Comment (poojasabnani) [reply comp.html]: Small suggestion:
In iframe components, except height and width, the following are common as far as I noticed:
'border:0; transform: scale(1); overflow:hidden;margin: 2rem 0 1rem; border-radius: 1rem' 

So only these properties can be moved to a common class (DRY principle).

**Comment by poojasabnani** [comment]
Hey @prerana1821, you're welcome! 
I do the code review by checking the "files changed" under the Pull Request on github only.
Cloning isn't required for PR review.

Trick: If you want to view this in VSCode format, just add 1s after 'github'.
Eg: your current PR link is: https://github.com/prerana1821/prekit/pull/2
to view like VSCode link is: https://github1s.com/prerana1821/prekit/pull/2

**Comment by purvasheth** [comment]
> Hey @prerana1821, you're welcome!
> I do the code review by checking the "files changed" under the Pull Request on github only.
> Cloning isn't required for PR review.
> 
> Trick: If you want to view this in VSCode format, just add 1s after 'github'.
> Eg: your current PR link is: #2
> to view like VSCode link is: https://github1s.com/prerana1821/prekit/pull/2

@py93, this is very helpful. Thanks for this. This will make giving good reviews much easier.

**Comment by prerana1821** [comment]
> Hi @prerana1821,
> Impressed 💯
> Your library is so pretty!
> 
> Somethings I noticed
> 
> * text contrast is less than it should be almost everywhere. Even on primary buttons. Please look into this. Your color sense is amazing! Keep the contrast in mind though.
> * your navigation can have an active style. Both top and side. The hover effect is pretty cool. But once the user selects something, it would be nice if he/she knows what is selected.
> 
> Pooja covered all the code-level comments :)

Hello Purva, Thanks a lot to you as well. 🙌❤ I have added the active class effect to both side & main navigation!

**Comment by poojasabnani** [comment]
> > Hey @prerana1821, you're welcome!
> > I do the code review by checking the "files changed" under the Pull Request on github only.
> > Cloning isn't required for PR review.
> > Trick: If you want to view this in VSCode format, just add 1s after 'github'.
> > Eg: your current PR link is: #2
> > to view like VSCode link is: https://github1s.com/prerana1821/prekit/pull/2
> 
> @py93, this is very helpful. Thanks for this. This will make giving good reviews much easier.

@purvasheth glad it's helpful!

File: comp.html (hunk 1 above)
Comment (prerana1821) [reply comp.html]: > Small suggestion:
> In iframe components, except height and width, the following are common as far as I noticed:
> 'border:0; transform: scale(1); overflow:hidden;margin: 2rem 0 1rem; border-radius: 1rem'
> 
> So only these properties can be moved to a common class (DRY principle).

Ohh yess, right. Thank you! I did it now 😊🙌

**Comment by prerana1821** [comment]
> Hey @prerana1821, you're welcome!
> I do the code review by checking the "files changed" under the Pull Request on github only.
> Cloning isn't required for PR review.
> 
> Trick: If you want to view this in VSCode format, just add 1s after 'github'.
> Eg: your current PR link is: #2
> to view like VSCode link is: https://github1s.com/prerana1821/prekit/pull/2

Woaah! This is insane!! 💯🔥 Can't thank enough!

**Review by RohitDhatrak — APPROVED** [review]
Good job Prerana great attention to detail.
I see that you have used pixels for setting the height and the width in many places. In most cases, pixels should be avoided as they are not responsive units. This video will help https://youtu.be/N5wpD9Ov_To

**Comment by prerana1821** [comment]
> Good job Prerana great attention to detail.
> I see that you have used pixels for setting the height and the width in many places. In most cases, pixels should be avoided as they are not responsive units. This video will help https://youtu.be/N5wpD9Ov_To

Hey @RohitDhatrak , Thanks a lot for you suggestions & I will remove fixed heights & widths. Thanks a lot for the video!

**Pull request merged**